billing_5c06ad9604286304a71cfa5b.md

```diff
@@ -1,26 +1,41 @@
 With Semaphore 2.0 you will only pay for what you actually used and you can
 customize your usage according to your team's needs.
 
+## Trial Period
+
+Once you sing up, your first organization will be automatically created based on your
+GitHub username and you will be in a 14-day trial period. During this period you will be 
+able to use [the default quota of machines](https://docs.semaphoreci.com/article/133-quotas-and-limits) completely free of charge. 
+
+Of course, you can easily scale up if you need a bigger capacity by going to **Billing > 
+See detailed insights… > Quota > Request upgrade…** or by reaching out to 
+[customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com) (please 
+include which type of machine you prefer).
+
+If you don't want to use trial period for the automatically created organization,
+please reach out to [support@semaphoreci.com](mailto:support@semaphoreci.com) and we will transfer
+it to another organization you own.
+
 ## Plans
 
-Once you sign up and create an organization, you are automatically on a
-**free plan**. With this plan, you will get $20 free credit every 30 days for
-the first organization you create on Semaphore 2.0. If needed, an owner of
-the organization can request a transfer of $20 free credit to another organization
-he owns by reaching out to [support@semaphoreci.com](mailto:support@semaphoreci.com).
+After your trial period expires, your organization will be blocked and you will be 
+prompted to choose **Free plan** or **Paid plan**.
+
+**Free plan** includes $10 free credit per calendar month and you will receive it on 
+the 1st of every month. On this plan, you will have parallelism 1 and you can use 
```

Comment:
`parallelism of 1` or `you will be able to run 1 job in parallel`

Reply:
It is not clear "what will I receive on the 1st every month".

Alternative:

> **Free plan** includes $10 free credit per calendar month. At the beginning of each month, you will receive a new $10 credit for your organization.